From 2f9455c8499345dcfa7553b4c1fd34e619be3ece Mon Sep 17 00:00:00 2001
From: Suman Anna <s-anna@ti.com>
Date: Fri, 8 Sep 2017 14:08:25 -0500
Subject: [PATCH 117/256] ARM: DRA7: Cleanup old pinctrl macros

Commit 6ae4c3efbd62 ("ARM: DRA7: Add pinctrl register definitions")
has added new macros for pinmux configuration in line with the kernel
definitions. Cleanup the old pinctrl macros from the common header
file so that they are not used by any new boards.

Signed-off-by: Suman Anna <s-anna@ti.com>
---
 arch/arm/include/asm/arch-omap5/mux_dra7xx.h | 14 --------------
 1 file changed, 14 deletions(-)

diff --git a/arch/arm/include/asm/arch-omap5/mux_dra7xx.h b/arch/arm/include/asm/arch-omap5/mux_dra7xx.h
index 5eed98ca27..55f49c7848 100644
--- a/arch/arm/include/asm/arch-omap5/mux_dra7xx.h
+++ b/arch/arm/include/asm/arch-omap5/mux_dra7xx.h
@@ -12,20 +12,6 @@
 
 #include <asm/types.h>
 
-#define FSC	(1 << 19)
-#define SSC	(0 << 19)
-
-#define IEN	(1 << 18)
-#define IDIS	(0 << 18)
-
-#define PTU	(1 << 17)
-#define PTD	(0 << 17)
-#define PEN	(1 << 16)
-#define PDIS	(0 << 16)
-
-#define WKEN	(1 << 24)
-#define WKDIS	(0 << 24)
-
 #define PULL_ENA		(0 << 16)
 #define PULL_DIS		(1 << 16)
 #define PULL_UP			(1 << 17)
-- 
2.13.6

